lib/datasource/maven/index.ts

```diff
@@ -178,6 +178,7 @@ async function getVersionsFromMetadata(
 
 type ArtifactsInfo = Record<string, boolean | null>;
 
+// istanbul ignore next
```

Comment:
We should add a test for this later

Reply:
Yeah, I'm not sure why it lost coverage but seems the previous test depended on the cache being set